autorest/codegen/models/parameter.py

```diff
@@ -195,7 +195,7 @@ def _default_value(self) -> Tuple[Optional[Any], str, str]:
         if self.multiple_media_types_type_annot:
             # means this parameter has multiple media types. We force default value to be None.
             default_value = None
-            default_value_declaration = "None"
+            default_value_declaration = ''
```

Comment:
![image](https://user-images.githubusercontent.com/70930885/131625595-d85c14b7-d8de-45e8-a57f-6e0f85d1c5a3.png)
@iscai-msft  How about set it with `''` instead of `"None"`?